services/archiver/archiver_test.go

```diff
@@ -132,3 +126,8 @@ func TestArchive_Basic(t *testing.T) {
 	assert.NotEqual(t, zipReq.GetArchiveName(), tgzReq.GetArchiveName())
 	assert.NotEqual(t, zipReq.GetArchiveName(), secondReq.GetArchiveName())
 }
+
+func TestErrUnknowArchiveFormat(t *testing.T) {
```

Comment:
`TestErrUnknowArchiveFormat` is supposed to be `TestErrUnknownArchiveFormat`, I presume.